content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,259 @@
+
+Customize columns
+~~~~~~~~~~~~~~~~~
+
+Additional columns can be added by clicking the :icon:`fa-cog` :guilabel:`Customize columns`. On the
+resulting popover window, click the drop-down menu for the :guilabel:`column` that should be
+changed.
+
+The following columns can be added for additional transactional information:
+
+ - :guilabel:`AvaTax calculated`: the amount of tax calculated by *AvaTax*.
+ - :guilabel:`Country`: the country this tax was calculated in; this is a text field.
+ - :guilabel:`Cust/vendor code`: the customer/vendor code in Odoo (e.g. `Contact18`).
+ - :guilabel:`Currency`: the standardized abbreviation for the currency the amount total is in.
+ - :guilabel:`Doc date`: the document's date of creation.
+ - :guilabel:`Doc status`: any of the following options, :guilabel:`All`, :guilabel:`Voided`,
+   :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Doc type`: any of the following selections, :guilabel:`All`, :guilabel:`Sales
+   Invoice`, :guilabel:`Purchase Invoice`, :guilabel:`Return Invoice`, :guilabel:`Inventory Transfer
+   Inbound Invoice`, :guilabel:`Inventory Transfer Outbound Invoice`, or :guilabel:`Customs
+   Invoice`.
+ - :guilabel:`Import ID`: represents the import ID of the document.
+ - :guilabel:`Last modified`: timestamp of the last time the document was modified.
+ - :guilabel:`Location code`: the location code used to calculate the tax, based on the delivery
+   address.
+ - :guilabel:`PO number`: the purchase order number.
+ - :guilabel:`Reference code`: the Odoo reference code (e.g. NV/2024/00003)
+ - :guilabel:`Region`: the region of the country,which varies based on the :guilabel:`Country`
+   selection.
+ - :guilabel:`Salesperson code`: the numeric ID of the user assigned to the sales order in Odoo.
+ - :guilabel:`Tax date`: the month/day/year of the tax calculation.
+ - :guilabel:`Tax override type`: where an exemption would appear, should there be none, the field
+   populates with :guilabel:`None`.
+
+To add a new *column* click the :icon:`fa-plus` :guilabel:`Column`.
+
+.. seealso::
+   For more information on *AvaTax* transactions, refer to this Avalara documentation: `Transactions
+   <https://community.avalara.com/support/s/document-item?language=en_US&bundleId=qvv1656594440497&topicId=transactions.html&_LANG=enus>`_.
+
+Import-export
+-------------
+
+While on the :ref:`avalara_portal/transactions`, click :icon:`fa-download` :guilabel:`Import
+transactions` or :icon:`fa-upload` :guilabel:`Export transactions` to import or export transactions.
+
+Reports
+-------
+
+To access reporting, navigate to the :menuselection:`Reports` link in the top menu of the Avalara
+management console. Next, select from one of the available reporting tabs: :guilabel:`Transactions
+reports`, :guilabel:`Liability & tax return reports`, or :guilabel:`Exemption reports`.
+
+.. seealso::
```

Comment:
the seealso:: admonition is traditionally reserved for additional links, since there are no links in this I suggest using a different admonition block :)
```suggestion
.. tip::
```